day_two_guide/topics/controller_role_verification.adoc

```diff
@@ -12,31 +12,39 @@ hosts. The service runs in active/passive mode, meaning it should only be
 running on one master at any time.
 
 The {product-title} controllers execute a procedure to choose which host runs
-the service. The current running value is stored in the etcd cluster within the `/openshift.io/leases/controllers` key.
+the service. The current running value is stored in an annotation in a special
+`configmap` stored in the `kube-system` project.
 
 Verification of the master host actively running the
 `atomic-openshift-master-controllers.service` service is performed by running
-the following command on any of the master nodes:
+the following command as `cluster-admin`:
 
 ----
-# etcdctl2 get /openshift.io/leases/controllers
-master-0fz9x9k2
+$ oc get -n kube-system cm openshift-master-controllers -o yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    control-plane.alpha.kubernetes.io/leader: '{"holderIdentity":"master-ose-master-0.example.com-10.19.115.212-dnwrtcl4","leaseDurationSeconds":15,"acquireTime":"2018-02-17T18:16:54Z","renewTime":"2018-02-19T13:50:33Z","leaderTransitions":16}'
+  creationTimestamp: 2018-02-02T10:30:04Z
+  name: openshift-master-controllers
+  namespace: kube-system
+  resourceVersion: "17349662"
+  selfLink: /api/v1/namespaces/kube-system/configmaps/openshift-master-controllers
+  uid: 08636843-0804-11e8-8580-fa163eb934f0
 ----
 
-The command outputs an identifier of the master host running the controller
-service, but currently this identifier is generated randomly when the service is
-executed. To get the ID of every master search the
-`atomic-openshift-master-controllers.service` logs for a master host ID:
+The command outputs the current master controller in the
+`control-plane.alpha.kubernetes.io/leader` annotation, within the
+`holderIdentity` property as:
 
 ----
-I1130 11:40:56.778373   30138 leaderelection.go:127] Attempting to acquire controller lease as master-8sn5gpzn, renewing every 30s
+master-<hostname>-<ip>-<8_random_characters>
 ----
 
-In order to find the identifiers of the master nodes, log in to each master node
-and run:
+In order to properly filter the hostname, the following command can be used:
```

Comment:
Not too sure if just using "filter" is enough information. Perhaps:

Find the identifiers of the master host by filtering the hostname, using the following:

Does that adequately sum up what's happening?

Reply:
I think I've fixed it, thx for the suggestion!